### ISSUE TITLE
Type CoreNextpageBlockAttributes must define one or more fields.

### ISSUE THREAD

**Issue by ktw-studio** [pristas-peter/wp-graphql-gutenberg#35]
Hey, 

I am working on the project of mine and I get the following error on clean WP installation with the latest version plugin. I use gatsby as my frontend application. I wonder if you have any clue what could cause it and how can I help to resolve the issue? I am more than happy to participate with some code if you can point me into the right direction.

"Type CoreNextpageBlockAttributes must define one or more fields."

Martin

**Comment by pristas-peter** [comment]
it seems like your query is missing selection on CoreNextpageBlockAttributes like 
```
on CoreNextpageBlockAttributes {
  something
someOtherField
}
```

**Comment by willem-aart** [comment]
Getting the exact same error. My application doesn't do anything with CoreNextPageBlock, so I guess something goes wrong in WP or in this plugin?

**Comment by pristas-peter** [comment]
Fixed in e179435655c239703eb25589a1b2acf03ca4b129
